LibraryApp-main/PersonalLibraryApp/PersonalLibraryApplication/Models/BooksRead.cs

```diff
@@ -1,45 +1,39 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace PersonalLibraryApp
-{
-    internal class BooksRead: Book
-    {
-        
-
-        bool Owned {  get; set; }
-        DateTime DateFinished { get; set; }
-        String Review {  get; set; }
-        public BooksRead(string bookId, string title, string description, string authorLastName) : base(bookId, title, description, authorLastName)
-        {
-            BookId = bookId;
-            Title = title;
-            Description = description;
-            AuthorLastName = authorLastName;
-        }
-
-        public BooksRead(bool owned, DateTime dateFinished, string review)
-        {
-            Owned = owned;
-            DateFinished = dateFinished;
-            Review = review;
-        }
-        public BooksRead(string bookId, string title, string description, string authorLastName, bool owned, DateTime dateFinished, string review) : base(bookId, title, description, authorLastName)
-        {
-            BookId = bookId;
-            Title = title;
-            Description = description;
-            AuthorLastName = authorLastName;
-            Owned = owned;
-            DateFinished = dateFinished;
-            Review = review;
-        }
-
-        public BooksRead()
-        {
-        }
-    }
-}
+﻿namespace PersonalLibraryApplication.Models
+{
+    internal class BooksRead: Book
```

Comment:
As an aside: Reconsider your use of abstraction on classes like this.

You are inheriting Book, when it should be an a Book Object set as a property instead.

So instead of `internal class BooksRead : Book`

You would have 
```
internal class BooksRead
{
    ...
    public Book Book { get; set; }
    public bool Owned { get; set; }
    ...
}
```